mmcls/models/heads/linear_head_gcvit.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
This new head seems similar with `LinearClsHead`, maybe you can use the `LinearClsHead` directly.